numpy/__init__.py

```diff
@@ -185,8 +185,12 @@ def pkgload(*packages, **options):
     pkgload.__doc__ = PackageLoader.__call__.__doc__
 
     from .testing import Tester
-    test = Tester().test
-    bench = Tester().bench
+    if '.dev0' in __version__:
+        test = Tester(raise_warnings="develop").test
+        bench = Tester(raise_warnings="develop").bench
+    else:
+        test = Tester(raise_warnings="release").test
+        bench = Tester(raise_warnings="release").bench
```

Comment:
It's not so much that line count is at a premium, as that the point of DRY is to keep things simple and obvious and easy to modify, so making things more complicated in order to make them DRY is a dubious win :-)